src/rejectdups-rust/src/lib.rs

```diff
@@ -0,0 +1,500 @@
+use byteorder::{ByteOrder, LittleEndian};
+use solana_sdk::{
+    account_info::AccountInfo, entrypoint, entrypoint::ProgramResult, info,
+    program_error::ProgramError, account_info::next_account_info, pubkey::Pubkey,
+    hash::Hasher, 
+    sysvar::{self, Sysvar}, rent::Rent,
+};
+use std::mem;
+
+
+pub fn create_address_with_seed(
+    base: &Pubkey,
+    seed: &str,
+    program_id: &Pubkey,
+) -> Pubkey {
+    let mut hasher = Hasher::default();
+    hasher.hashv(&[base.as_ref(), seed.as_ref(), program_id.as_ref()]);
+    Pubkey::new( hasher.result().as_ref() )
+}
+
+// Declare and export the program's entrypoint
+entrypoint!(process_instruction);
+
+// Program entrypoint's implementation
+fn process_instruction<'a>(
```

Comment:
Do you need the lifetime `'a`?

Reply:
seems not. copied it from here: https://github.com/solana-labs/example-helloworld/blob/01c6822ef14a777cad45f48d6809e52bc22f2f42/src/program-rust/src/lib.rs#L18

Reply:
ah, thanks for pointing that out, I'll remove those lifetimes from the helloworld example